### PR TITLE
Doc: Issue 41: Add short description of each implemented LIC unit test

### LINKED ISSUE #41: Ensure that all test cases are appropriately documented in natural language
We must ensure that:
> [t]he contract that is tested is clearly formulated in natural language

for each unit test, according to the [grading criteria](https://canvas.kth.se/courses/31884/assignments/185707).

Every unit test should have a short description of which particular functionality it is meant to test.

### PR DESCRIPTION
**Pull request by siberov**
Each implemented LIC unit test has been equipped with a description
of it's intended outcome, as well as the reason for this outcome.

Closes #41

### CODE REVIEW

**Comment by KatrinaLiang** [comment]
Tests for LIC6 are in the main branch. Merge main and add comments for these tests :)

File: src/test/java/decide/CMVTest.java [hunk 1]
```diff
@@ -182,6 +210,9 @@ void lic3FailTest() {
         assertFalse(cmv.get(3));
     }
 
+    /**
+     * Test that LIC 3 is false when the three points doesn't form a triangle.
+     */
     @Test
     @DisplayName("LIC 3 Illegal triangle")
     void lic3IllegalTriangleTest() {
```
Comment (alevarn) [inline src/test/java/decide/CMVTest.java:218]: Maybe rephrase this to "LIC 3 Triangle zero area" instead of "Illegal triangle".

**Review by alevarn — APPROVED** [review]
LGTM!

**Comment by KatrinaLiang** [comment]
Looking good
